internal/firewall/validation_test.go

```diff
@@ -58,8 +48,58 @@ func TestValidateIPDiag(t *testing.T) {
 			}
 
 			if test.err != nil {
-				assert.Equal(t, err, test.err)
+				assert.Equal(t, test.err, err)
 			}
 		})
 	}
 }
+
+func Test_normalizeIP(t *testing.T) {
```

Comment:
I believe we are missing the badly formatted IP address test case, should we are it ?

Reply:
Good catch, added a testcase for this.